### DISCUSSION TITLE
What about Orientdb and log4j vuln?

### DISCUSSION THREAD

**Discussion by frbayart** [orientechnologies/orientdb#9737] [Q&A] [answered]
I suppose everyone have heard about  Log4j2 CVE-2021-44228. What this the impact on Orientdb?

If any do you plan to release versions to fix it?

**Comment by andrii0lomakin** [comment, marked as answer]
Hi, we do not have log4j dependencies :-)

**Comment by frbayart** [comment]
Ok I see it is only used in `test` part. Thx
